### ISSUE TITLE
what does n_class of logits in gumbel_softmax represent?

### ISSUE THREAD

**Issue by turmeric-blend** [chaoshangcs/GTS#3]
Hi, what does [n_class](https://github.com/chaoshangcs/GTS/blob/208bc8bb5afbbdd1ea6cddbecbb9ccd0e9effdc7/model/pytorch/model.py#L29) of the logits input in gumbel_softmax function represent?

**Comment by chaoshangcs** [comment]
Hi. It represents the number of classes. Here the n_class is 2.

**Comment by turmeric-blend** [comment]
what are these classes?

**Comment by chaoshangcs** [comment]
It is the general binary classification. Each edge is either 0 or 1.

**Comment by turmeric-blend** [comment]
I see. Thanks.
